config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'results/index'
+
+  get 'result/index'
```

Comment:
I don't think you are using this bottom route, lets remove it.

Reply:
Done and done.